airbyte-integrations/connectors/source-stripe/source_stripe/source.py

```diff
@@ -483,6 +472,13 @@ def streams(self, config: MutableMapping[str, Any]) -> List[Stream]:
                 sub_items_attr="refunds",
                 **args,
             ),
+            UpdatedCursorIncrementalStripeSubStream(
+                name="payment_methods",
+                path=lambda self, stream_slice, *args, **kwargs: f"customers/{stream_slice['parent']['id']}/payment_methods",
+                parent=self.customers(**args),
+                event_types=["payment_method.*"],
```

Comment:
I would be wary of using star pattern matching for reasons like [this](https://github.com/airbytehq/airbyte/blob/75cee334ceda9f2b70e168db883abf4f52ecea21/airbyte-integrations/connectors/source-stripe/source_stripe/source.py#L249-L251).

It feels like there is no good solution here. Either we try the pattern matching and are at risk of breaking the stream or either we explicit every single events but when one is added, we face data loss. I'm not sure what is best here

Reply:
🤔 ,
thanks for pointing out, I _think_ we may consider issue `invoice.upcoming` as an exception and use pattern match here.

Reply:
I'm fine with considering this a one-off and see how it goes